awx/main/tests/data/inventory/plugins/ec2/files/aws_ec2.yml

```diff
@@ -0,0 +1,82 @@
+boto_profile: /tmp/my_boto_stuff
+compose:
+  ansible_host: public_ip_address
+  ec2_account_id: network_interfaces | json_query("[0].owner_id")
```

Comment:
`network_interfaces[0].owner_id` will negate the need for `json_query`

Reply:
You can use `owner_id` now that it was added to the default hostvars in https://github.com/ansible/ansible/commit/8b80a71f9ffc03ac5886a915c9215b4e9a86dba0